app/realty/views.py

```diff
@@ -1,2 +1,34 @@
 from django.shortcuts import render
 
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from .models import Flat
+from .serializers import FlatSerializer
+
+
+class AllFlatsApi(APIView):
+    def get(self, request):
+        data = Flat.objects.all()
+        return Response({'flats': FlatSerializer(data, many=True).data})
```

Comment:
Здесь так должно сработать
```suggestion
        return Response(data=FlatSerializer(data, many=True).data)
```